cirq/sim/clifford/stabilizer_state_ch_form.py

```diff
@@ -64,8 +66,32 @@ def bits(s):
             if val:
                 self._X(self.n - i - 1)
 
+    def _json_dict_(self):
+        return protocols.obj_to_dict_helper(
+            self,
+            ['n', 'initial_state', 'G', 'F', 'M', 'gamma', 'v', 's', 'omega'])
+
+    @classmethod
+    def _from_json_dict_(cls, n, initial_state, G, F, M, gamma, v, s, omega,
+                         **kwargs):
+        copy = StabilizerStateChForm(n, initial_state)
+
+        copy.G = G.copy()
+        copy.F = F.copy()
+        copy.M = M.copy()
+        copy.gamma = gamma.copy()
+        copy.v = v.copy()
+        copy.s = s.copy()
+        copy.omega = omega
+
+        return copy
+
+    def _value_equality_values_(self):
+        return (self.n, self.initial_state, self.G, self.F, self.M, self.gamma,
+                self.v, self.v, self.s, self.omega)
+
     def copy(self) -> 'cirq.StabilizerStateChForm':
-        copy = StabilizerStateChForm(self.n)
+        copy = StabilizerStateChForm(self.n, self.initial_state)
```

Comment:
After looking at this code: why is `initial_state` stored on the class? Like the other `initial_state` arguments it should be used to set up the initial state but the class's "data" should just be the current state. Can you do a git blame to figure out who we should tag with this question?

Reply:
@vtomole in this PR:
https://github.com/quantumlib/Cirq/pull/2129

It is used later on but only for display, here:
https://github.com/quantumlib/Cirq/blob/740ef6c8034b25a0d2e6b763600b9f8cc5f5c899/cirq/sim/clifford/stabilizer_state_ch_form.py#L87

I figured it was more correct to add it, because it might confuse someone who is debugging, although I don't think it would cause incorrect simulation results. Happy to keep it in this PR, add it to another PR, or just revert.